buildenv/jenkins/JenkinsfileBase

```diff
@@ -79,6 +79,9 @@ def setupEnv() {
 	env.KEEP_REPORTDIR = params.KEEP_REPORTDIR ? params.KEEP_REPORTDIR : false
 	SDK_RESOURCE = params.SDK_RESOURCE ? params.SDK_RESOURCE : "upstream"
 	env.AUTO_DETECT = params.AUTO_DETECT
+	env.EXTERNAL_CUSTOM_REPO=params.EXTERNAL_CUSTOM_REPO ? params.EXTERNAL_CUSTOM_REPO : ""
+	env.EXT_REPO_BRANCH=params.EXT_REPO_BRANCH ? params.EXT_REPO_BRANCH : "master"
+	env.EXT_TEST_CMD=params.EXT_TEST_CMD ? params.EXT_TEST_CMD : "mvn clean install"
```

Comment:
Could we have a common naming convention? 
EXTERNAL_TEST_REPO
EXTERNAL_TEST_BRNACH
EXTERNAL_TEST_CMD

Reply:
@lxia Sure. I will change all the variable names to the above both in this file and the external_custom target.

Reply:
@smlambert I think we need to  change the variable names in [https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder_Tweetest/](https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder_Tweetest/)?

Reply:
Updated Grinder_Tweetest with new names now

Reply:
Thanks @smlambert